stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/Android.kt

```diff
@@ -612,4 +613,19 @@ class Android {
                 }
         }
     }
+
+    /**
+     * @see <a href="https://getstream.io/nessy/docs/chat_docs/events/event_listening?language=kotlin">Listening for events</a>
+     */
+    class SyncHistory() : Fragment() {
+
+        fun getSyncHistory(chatClient: ChatClient) {
+            val cidList: List<String> = listOf("messaging:123")
+            val lastSeenExample = Date()
+
+            chatClient.getSyncHistory(cidList, lastSeenExample).enqueue { result ->
+                // Use result here.
```

Comment:
Let's align this with our other docs, and show the check for success, plus what type we received, e.g.

```kotlin
if (result.isSuccess) {
     val events: List<ChatEvent> = result.data()
} else {
     // Handle result.error()
}
```